### PR TITLE
MB-7063 Add domestic unpacking

### PR DESCRIPTION
**Pull request by monfresh**
## Description

This is a continuation of the pricing work we started last week. It adds the pricing details for the Domestic unpacking.

## Reviewer Notes

1. After running the setup steps below, go to http://officelocal:3000/devlocal-auth/login
2. Click on the "Login" button next to "tio_role@office.mil (PPM office)"
and choose the TIO user. 

2. Click on the payment request with the `PARAMS` Move Code.
3. Click on Review Service Items
4. Approve them all and click Approve payment
5. Once back on the payment requests overview page, click on "Domestic unpacking" to reveal the calculations

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_dev_e2e_populate
make server_run
make office_client_run
```

## Code Review Verification Steps

* [ ] If the change is risky, it has been tested in experimental before merging.
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] The requirements listed in [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely) have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
  * [ ] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-7063) for this change

## Screenshots

Screen Shot 2021-03-30 at 1 07 02 PM

### CODE REVIEW

**Review by amanda5iu — APPROVED** [review]

File: src/components/Office/ServiceItemCalculations/helpers.test.js [hunk 1]
```diff
@@ -199,8 +199,29 @@ describe('makeCalculations', () => {
...
-    expect(result).toEqual([]);
+    const result = makeCalculations('DUPK', 99999, testParams.DomesticUnpacking);
+    expect(result).toEqual([
+      {
+        value: '85 cwt',
+        label: 'Billable weight (cwt)',
+        details: ['Shipment weight: 8,500 lbs', 'Estimated: 8,000 lbs'],
+      },
+      {
+        value: undefined,
```
Comment (duncan-truss) [inline src/components/Office/ServiceItemCalculations/helpers.test.js:210]: Do you want to pass a PriceRateOrFactor param in the test data to make sure this gets populate when it becomes available?

File: src/components/Office/ServiceItemCalculations/serviceItemTestParams.js [hunk 2]
```diff
@@ -248,7 +248,7 @@ const ServicesScheduleOrigin = {
   key: 'ServicesScheduleOrigin',
   origin: 'SYSTEM',
   paymentServiceItemID: '28039a62-387d-479f-b50f-e0041b7e6e22',
-  type: 'STRING',
+  type: 'INT',
```
Comment (duncan-truss) [inline src/components/Office/ServiceItemCalculations/serviceItemTestParams.js:251]: Nice catch it's a Stringy number

File: src/constants/serviceItems.js [hunk 3]
```diff
@@ -18,7 +18,9 @@ const SERVICE_ITEM_PARAM_KEYS = {
   IsPeak: 'IsPeak',
   ServiceAreaOrigin: 'ServiceAreaOrigin',
   ServicesScheduleOrigin: 'ServicesScheduleOrigin',
+  ServicesScheduleDest: 'ServicesScheduleDest',
   RequestedPickupDate: 'RequestedPickupDate',
+  RequestedDeliveryDate: 'RequestedDeliveryDate',
```
Comment (duncan-truss) [inline src/constants/serviceItems.js:23]: Will we ever need this for another service item or we can get rid of it?

**Review by duncan-truss — APPROVED** [review]
Looks good to me, could you add it to Storybook too please?

**Review by Ryan-Koch — APPROVED** [review]

**Pull request merged**